src/sass/components/_post-box.scss

```diff
@@ -1,6 +1,13 @@
+a.not-underline {
```

Comment:
A better place for tag selectors is at the end of file `style.scss` - there is also `body` selector. Please move it there.